pkg/utils/helpers.go

```diff
@@ -63,6 +64,12 @@ func HrToYaml(hr fluxhelmv2beta1.HelmRelease) string {
 	return string(b)
 }
 
+func HrToAnyStringPtr(hr *fluxhelmv2beta1.HelmRelease) *v1alpha13.AnyString {
+	yaml := HrToYaml(*hr)
+	base64 := base64.StdEncoding.EncodeToString([]byte(yaml))
+	return ToAnyStringPtr(base64)
+}
+
```

Comment:
The name doesn't capture that we are converting it into base64 before returning it as an `AnyString` value.
Maybe `HrToB64AnyStringPtr()`?

or we make this is a generic function,
```go
func ObjToAnyStringPtr(obj runtime.Object, transformFn func(obj runtime.Object) string) *v1alpha13.AnyString {
  return ToAnyStringPtr(transformFn(obj))
}
  
// Invocation
ObjToAnyStringPtr(hr, func(obj runtime.Object) string {
  	b, err := yaml.Marshal(hr)
	if err != nil {
		return ""
	}
	return string(b)
})
```

or something along those lines.